src/main/kotlin/com/pubnub/api/models/consumer/history/PNFetchMessage.kt

```diff
@@ -38,7 +39,8 @@ data class PNFetchMessageItem(
     val meta: JsonElement?,
     val timetoken: Long,
     val actions: Map<String, Map<String, List<Action>>>? = null,
-    val messageType: HistoryMessageType?
+    val messageType: HistoryMessageType?,
+    val error: PubNubError? = null
```

Comment:
add documentation (and in the other class) that explains that this can take a value of `CRYPTO_IS_CONFIGURED_BUT_MESSAGE_IS_NOT_ENCRYPTED` (and in what situation) or null